src/qt/optionsdialog.cpp

```diff
@@ -45,6 +45,8 @@ OptionsDialog::OptionsDialog(QWidget *parent, bool enableWallet) :
     ui->threadsScriptVerif->setMinimum(-GetNumCores());
     ui->threadsScriptVerif->setMaximum(MAX_SCRIPTCHECK_THREADS);
 
+    ui->networkPort->setValidator(new QIntValidator(1, 65535, this));
```

Comment:
@tulip0 That is not universally true. I think Windows allows non-admin to bind them, and on Linux there are capabilities to allow binding them.